templates/nfs-install.sh

```diff
@@ -0,0 +1,12 @@
+apt install -y nfs-kernel-server
+mkdir -p /srv/domino
+mkfs.ext4 /dev/disk/by-id/google-nfs
+UUID=$(blkid /dev/disk/by-id/google-nfs -s UUID -o value)
+echo UUID=$UUID /srv/domino ext4 defaults 0 2 >> /etc/fstab
+systemctl daemon-reload
+mount /srv/domino
+chmod 777 /srv/domino
+echo '/srv/domino 10.0.0.0/255.0.0.0(rw,async,no_root_squash)' >> /etc/exports
+systemctl enable nfs-kernel-server --now
+sleep 5
+/etc/init.d/nfs-kernel-server restart
```

Comment:
For some reason, the exports file didn't seem to be respected when I first tested this, but worked on restart. Will re-test this before merge, since I don't think it should be necessary.

Reply:
It is necessary. :-/ Leaving it in, but adding a comment.